code-scanning/codeql.yml

```
@@ -24,6 +24,9 @@ jobs:
   analyze:
     name: Analyze
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      security-events: write
```

```suggestion
      security-events: write
      actions: read
```
@hubwriter is correct that we should include this additional permission for the codeql workflow.

---

Thanks! Fixed